nix/ignis.nix

```
@@ -1,6 +1,45 @@
-{ self, fetchFromGitHub, pkgs, version ? "git", ... }:
+{
+  lib,
+  dart-sass,
+  fetchFromGitHub,
+  git,
+  glib,
+  gnome-bluetooth,
+  gst_all_1,
+  gtk4,
+  gtk4-layer-shell,
+  gobject-introspection,
+  libpulseaudio,
+  librsvg,
+  makeWrapper,
+  meson,
+  networkmanager,
+  ninja,
+  pipewire,
+  pkg-config,
+  python3,
+  stdenv,
+  rev ? "dirty",
+  version ? "git",
+  ...
+}:
+
 let
-  inherit (pkgs.lib) concatStringsSep;
+  inherit (builtins) concatStringsSep;
+
+  python = python3.withPackages (
+    ps: with ps; [
+      certifi
+      charset-normalizer
+      click
+      idna
+      loguru
+      markupsafe
+      pycairo
+      pygobject3
+      urllib3
```

Since this PR is some kind of refactoring, let's remove extra dependencies here.
```suggestion
      click
      loguru
      pycairo
      pygobject3
```